README.md

```diff
@@ -1,5 +1,8 @@
 # Andromeda
-Andromeda is a website that allows a user to perform dimensional reduction on an uploaded CSV file.
+This Andromeda package combines the data visualization tool (andromeda) with an option to fetch data from iNaturalist observations and pair them with satellite RGB and landcover data for analysis with andromeda. Andromeda uses a multi-dimensional scaling algorithm to display data in clusters on a 2-dimensional plot. It is interactive, allowing the user to choose image representations as their points and rearrange the clusters in a manner they find appropriate. This new grouping can be fed back into the algorithm to determine the weighting on provided features that most accurately describe it. Alternatively, the user can decide to increase the weight applied to a particular feature and see how that alters the clustering (they are defaulted to equal weight).
```

Comment:
Good start. I'm wondering if the visualization aspect should be front and center and the fetch data part is an additional feature. All the different uses of Andromeda vs andromeda are rather confusing.

Reply:
Also can we have an image of the visualization page in the README.

Reply:
Agreed; I wanted to get the conversation started with something more descriptive up here. 

I'll get a picture up!